sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -148,7 +165,10 @@ class DDLSuite extends QueryTest with SharedSQLContext with BeforeAndAfterEach {
         try {
           val dbNameWithoutBackTicks = cleanIdentifier(dbName)
           val location =
-            System.getProperty("java.io.tmpdir") + File.separator + s"$dbNameWithoutBackTicks.db"
+            catalog.makeQualifiedPath(
```

Comment:
it'd be great to make the testing of qualified path more explicit, e.g. by having a test case dedicated to it.

The remaining test cases can just check the path is part of the intended qualified path.